client/src/pages/SplashScreen/SplashScreen.tsx

```diff
@@ -0,0 +1,27 @@
+
+import { useState, useEffect } from "react";
+import { ScreenOne, ScreenTwo } from "../../components";
+import { useNavigate } from 'react-router-dom';
+
+export const SplashScreen = () => {
+    const [next_screen, setNext_screen] = useState("first_screen")

```

Comment:
Please name the return values from useState in camelCase. `[nextScreen, setNextScreen] = ...`